politeiawww/cmd/piwww/userverifypayment.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2017-2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package main
+
+import "github.com/decred/politeia/politeiawww/cmd/shared"
+
+// userVerifyPaymentCmd checks on the status of the logged in user's
+// registration payment.
+type userVerifyPaymentCmd struct{}
+
+// Execute executes the verify user payment command.
+func (cmd *userVerifyPaymentCmd) Execute(args []string) error {
+	vupr, err := client.VerifyUserPayment()
```

Comment:
Client method namings are consistent with the types `www` api describes, so I thought it was better to just leave them to match the types.